src/Core/Domain/Product/Command/UpdateProductCategoriesCommand.php

```diff
@@ -28,9 +28,13 @@
 
 namespace PrestaShop\PrestaShop\Core\Domain\Product\Command;
 
+use LogicException;
 use PrestaShop\PrestaShop\Core\Domain\Category\ValueObject\CategoryId;
 use PrestaShop\PrestaShop\Core\Domain\Product\ValueObject\ProductId;
 
+/**
+ * Updates product-category associations
+ */
 class UpdateProductCategoriesCommand
```

Comment:
Avoid generic terms like "update"

```suggestion
class SetAssociatedProductCategoriesCommand
```